<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarExceptionDialog</class>
 <widget class="QDialog" name="CalendarExceptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="3">
    <widget class="QSpinBox" name="edtEndYear">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;День</string>
     </property>
     <property name="buddy">
      <cstring>edtDay</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblYear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Год с </string>
     </property>
     <property name="buddy">
      <cstring>edtBegYear</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="edtBegYear">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblDow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Числить как</string>
     </property>
     <property name="buddy">
      <cstring>cmbDow</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chkEndYearDiffer">
     <property name="text">
      <string>по</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="edtDay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMonth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Месяц</string>
     </property>
     <property name="buddy">
      <cstring>cmbMonth</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QComboBox" name="cmbMonth">
     <item>
      <property name="text">
       <string>Январь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Февраль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Март</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Апрель</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Май</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Июнь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Июль</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Август</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Сентябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Октябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ноябрь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Декабрь</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QComboBox" name="cmbDow">
     <item>
      <property name="text">
       <string>Понедельник</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вторник</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Среда</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Четверг</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Пятница</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Суббота</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Воскресенье</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>edtBegYear</tabstop>
  <tabstop>chkEndYearDiffer</tabstop>
  <tabstop>edtEndYear</tabstop>
  <tabstop>cmbMonth</tabstop>
  <tabstop>edtDay</tabstop>
  <tabstop>cmbDow</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalendarExceptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalendarExceptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEndYearDiffer</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndYear</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
